### ISSUE TITLE
关于lvis1.0中计算d

### ISSUE THREAD

**Issue by dmy1997** [KaihuaTang/Long-Tailed-Recognition.pytorch#21]
def update_embed(self, targets, gt_label):
        if self.training:
            # remove background
            with torch.no_grad():
                fg_target = targets[gt_label > 0].clone().detach().mean(0, keepdim=True)
                self.causal_embed = self.MU * self.causal_embed + fg_target
        return

新版mmdet中0-(num_classes-1)才是正样本，上面gt_label > 0是不是错了？

**Comment by KaihuaTang** [comment]
有可能，我把inference时候的这个问题给修复了
![image](https://user-images.githubusercontent.com/24402685/98889915-cd8aed00-24d5-11eb-9e89-207f0103064f.png)

但是计算D有没有这个问题我好像没检查。你可以试试改下看能不能多提升点

**Comment by dmy1997** [comment]
好的，感谢
